src/kvstore/kvstore_dist_server.h

```diff
@@ -434,7 +445,7 @@ class KVStoreDistServer {
           gradient_compression_->Dequantize(recved, &merged.array, 0);
         } else {
           gradient_compression_->Dequantize(recved, &decomp_buf, 0);
-          merged.array += decomp_buf;
```

Comment:
Change NDArray's += operator to use kernelLaunch instead of adding MergeReceived here